backend/main.py

```diff
@@ -35,12 +35,15 @@ async def google(request):
 async def mayonnaise(request):
     return RedirectResponse('https://www.youtube.com/embed/9K2Y-rfUy_4?autoplay=1&mute=1&controls=0', 303)
 
+async def cos(request):
+    return JSONResponse({'cos': str(request.__dict__)})
```

Comment:
do not expose meta data about application that can be security leak (You just expose that backend use Starlette and uvicorn)